### DISCUSSION TITLE
Welcome to Blue-topaz-examples Discussions!

### DISCUSSION THREAD

**Discussion by cumany** [cumany/Blue-topaz-examples#17] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cgonealtx** [comment]
The Advanced Blue Topaz vault has some great ideas but have no idea how to implement any of it since it's all in Oriental language. The clock on the right is superb and would love to put that in my vault but I've wasted countless hours trying to figure out or what thing say in order to utilize it. Frustrating :(
